deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# -------- Build Settings -------- #
+# directory on target server
+DEPLOY_DIR="./latest"
+# use to install dependencies without disruption
+TEMP_DIR="./staged"
+# server to connect to
+TARGET="hanbot@hanbot.elviswolcott.com"
+# name to give job (mostly for debugging on server)
+JOB_NAME="hanbot"
+
+# -------- Append to .env -------- #
+echo "
+TRAVIS_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER}
+TRAVIS_COMMIT=${TRAVIS_COMMIT}
+" >> ".env"
+
+# --------- Deploy Steps --------- #
+# transfer the build to the target
+rsync -r --delete-after --quiet --exclude-from=".deployignore" "${TRAVIS_BUILD_DIR}/" "${TARGET}:${TEMP_DIR}"
+# restart pm2 on the target
+ssh "${TARGET}" 'bash -s' <<RESTART
+cd "${TEMP_DIR}"
+npm install --only=production
```

Comment:
Is it problematic to use `yarn` in production? (I could see it being an issue for installation reasons.) If so, let's just switch to `npm` for everything (which, note to self, means that I need to revert [this commit](https://github.com/ariporad/hanbot/commit/99f0e074a2b84064d47c691cf0cfbbe10d471f62)).

Reply:
I can either commit an up to date package-lock and switch everything over to npm or get yarn installed on the server. Both shouldn't take more than a few minutes.

Reply:
Let's switch to npm. Yarn can be a PITA to install and NPM is everywhere.